rusk/src/lib/http/chain.rs

```diff
@@ -124,4 +127,30 @@ impl RuskNode {
 
         Ok(ResponseData::new(serde_json::to_value(&info)?))
     }
+
+    /// Calculates the average gas price of transactions in the mempool.
+    ///
+    /// It retrieves, at maximum, the top 100 transactions sorted by descending
+    /// order by the gas price, and calculates the average gas price. If there
+    /// are no transactions available in the mempool, it defaults to a gas price
+    /// of 1.
+    async fn get_gas_price(&self) -> anyhow::Result<ResponseData> {
+        let average_gas_price =
+            self.db().read().await.view(|t| -> anyhow::Result<u64> {
+                let (total, count) = t
+                    .get_txs_sorted_by_fee()?
+                    .take(100) // TODO: Figure out a sane default
+                    .map(|tx| tx.inner.fee().gas_price)
+                    .fold((0u64, 0u64), |(total, count), gas_price| {
+                        (total + gas_price, count + 1)
+                    });
+
+                match count {
+                    0 => Ok(1),
+                    _ => Ok(total / count), // TODO: Proper rounding up
```

Comment:
Not sure of what is meant by "proper rounding up", but if it is what I think you can use something like:

```rust
// If only unsigned
a / b + (a % b)
// If signed
a / b + (a % b).signum()
```

Reply:
> Not sure of what is meant by "proper rounding up", but if it is what I think you can use something like:
> 
> ```rust
> // If only unsigned
> a / b + (a % b)
> // If signed
> a / b + (a % b).signum()
> ```

It rounds down now, I want to use `div_ceil` instead:
```rust
_ => Ok(total.div_ceil(count)),
```